client/my-sites/sidebar/index.jsx

```diff
@@ -646,9 +604,6 @@ export class MySitesSidebar extends Component {
 	}
 
 	trackWooCommerceNavItemClick = ( menuItemName, experience, plan ) => {
```

Comment:
I think we can remove this whole function.

Reply:
I wasn't sure if the calypso_woocommerce_nav_item_click was being used and that's why I didn't remove it initially.  I'll delete the function.

Reply:
Done.